server/charts/historian/templates/historian-configmap.yaml

```diff
@@ -20,6 +20,7 @@ data:
         },
         "requestSizeLimit": "1gb",
         "riddler": "{{ .Values.historian.riddler }}",
+        "ignoreEphemeralFlag": "{{ .Values.historian.ignoreEphemeralFlag }}",
```

Comment:
Is this a boolean? If yes, enclosing it in quotes will break the logic. This should be:
`"ignoreEphemeralFlag": {{ .Values.historian.ignoreEphemeralFlag }},`

Reply:
Good catch. Fixed it.